Bottom: dc7515afd3bafeb8189262f26559c562c443a6ae
Top:    dc17aa74e00105ddfa301688cff184c84925a55c
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-05-28 00:08:32 -0300

Refresh of call-exprs.patch

---

diff --git a/include/gcc-cp-fe.def b/include/gcc-cp-fe.def
index d9e4289..ce73496 100644
--- a/include/gcc-cp-fe.def
+++ b/include/gcc-cp-fe.def
@@ -533,8 +533,6 @@ GCC_METHOD3 (gcc_expr, call_expr,
 	     gcc_expr,			      /* Argument CALLABLE.  */
 	     int /* bool */,		  /* Argument QUALIFIED_P.  */
 	     const struct gcc_cp_function_args *) /* Argument ARGS.  */
-	     
-/* FIXME: function call operations?  ctor/dtor?  new/delete?  */
 
 /* Return the type of the gcc_expr OPERAND.
    Use this for decltype.
